poly-commitment/src/ipa.rs

```diff
@@ -584,11 +583,6 @@ where
         })
     }
 
-    /// This function commits a polynomial using the SRS' basis of size `n`.
```

Comment:
Not sure if this is intentional, but it seems like the docs for these functions have been removed.

Reply:
Oh, it looks like you're moving the comments to the lib instead